index.html

```diff
@@ -10711,12 +10711,21 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
-                <div class="property" id="aria-details">
+		<div class="property" id="aria-details">
 			<pdef>aria-details</pdef>
 			<div class="property-description">
-				<p>Identifies the <a>element</a> that provides a detailed, extended description for the <a>object</a>. See related <pref>aria-describedby</pref>.</p>
-				<p>The <code>aria-details</code> attribute references a single element that provides more detailed information than would normally be provided by <pref>aria-describedby</pref>. It enables <a>assistive technologies</a> to make users aware of the availability of an extended description as well as navigate to it. Authors SHOULD ensure the element referenced by <code>aria-details</code> is visible to all users.</p>
-				<p>Unlike elements referenced by <code>aria-describedby</code>, the element referenced by <code>aria-details</code> is not used in either the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_name">Name Computation</a> or the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description">Description Computation</a> as defined in the Accessible Name and Description specification [[ACCNAME-1.1]]. Thus, the content of an element referenced by <code>aria-details</code> is not flattened to a string when presented to assistive technology users. This makes <code>aria-details</code> particularly useful when converting the information to a string would cause a loss of information or make the extended description more difficult to understand.</p>
+				<p>Identifies one or more elements that provide additional, extended information about the <a>object</a>.</p>
+				<p>The additional information can be:</p>
+				<ul>
+					<li>A description or general annotation (this is the default). See related <pref>aria-describedby</pref>.</li>
```

Comment:
As discussed at https://www.w3.org/2019/09/16-aria-minutes.html#item02 I think we need to stop using the word description in aria-details prose https://www.w3.org/2019/09/16-aria-minutes.html#item02

Reply:
What if we were clear that a description is only one of many potential types of aria-details/annotations? If we avoid saying description at all, how would authors know that they can use aria-details for a description that can be navigated to?

Reply:
I think it's clear that a description is only one of many potential types from the fact that it is followed by comment and definition and footnote and endnote. 

I like "general annotation." 

As I alluded to above, I don't see the benefit of moving the "See related" here. Please explain.

Reply:
Do we want to put an authors SHOULD and a role for the general annotation case?

Context: In my overall comment/review, I asked for the requirement that authors name the object which has the details. This makes it possible for screen readers to say things like "has details in the requirements list." Rather than just "has details" or "has details in list." Then I subsequently realized that we have plenty of objects/roles which cannot be named....

Reply:
What would the role be if the aria-details is a general annotation or description? I don't think there is always an appropriate role for a general annotation.

Also, while I think the author should use a role if there is an appropriate one, but that's a general ARIA truth anyway and is just adding more verbage. 

I move the "See related" back to where it was, but I actually think it was better where I moved it. Because aria-describedby is only related and only makes potential sense for a general description or annotation, not for a comment, footnote, etc.

Reply:
Given we agreed that we wouldn't be pushing authors to name the objects, my suggestion to push the role is obsolete.

As for the see related, let's see what others think. But for the most part I believe our "see related" instances are in the first paragraph/short definition of the related thing.

Reply:
Is this good as it stands now?